### PR TITLE
Updated medication use selection feedback

### PR DESCRIPTION
**Pull request by issyzac**

### CODE REVIEW

File: opensrp-chw/src/at/assets/json.form/adolescent_hv_medication_in_use_sw.json [hunk 1]
```diff
@@ -0,0 +1,587 @@
+{
+  "count": "1",
+  "encounter_type": "Tembeleo la Kijana Balehe - Utumiaji wa madawa",
```
Comment (gosso22) [inline opensrp-chw/src/at/assets/json.form/adolescent_hv_medication_in_use_sw.json]: I dont remember how this works, do we need to translated the encounter type? I think from the data's perspective, we are better off having the same names for encounter types

Comment (issyzac) [reply opensrp-chw/src/at/assets/json.form/adolescent_hv_medication_in_use_sw.json]: 🤔 I remember that, I think we can keep it the same name. Let me change that and update

File: opensrp-chw/src/at/assets/json.form/anc_hv_medication_in_use_sw.json [hunk 2]
```diff
@@ -0,0 +1,811 @@
+{
+  "count": "1",
+  "encounter_type": "Tembeleo la mama mjamzito - Matumizi ya madawa",
```
Comment (gosso22) [inline opensrp-chw/src/at/assets/json.form/anc_hv_medication_in_use_sw.json]: Same comment here

File: opensrp-chw/src/at/assets/json.form/child_hv_medication_in_use_sw.json [hunk 3]
```diff
@@ -0,0 +1,650 @@
+{
+  "count": "1",
+  "encounter_type": "Tembeleo la mtoto - Matumizi ya madawa",
```
Comment (gosso22) [inline opensrp-chw/src/at/assets/json.form/child_hv_medication_in_use_sw.json]: Same comment here

File: opensrp-chw/src/at/assets/json.form/pnc_hv_medication_in_use_sw.json [hunk 4]
```diff
@@ -0,0 +1,587 @@
+{
+  "count": "1",
+  "encounter_type": "Tembeleo na mama aliyejifungua - Matumizi ya madawa",
```
Comment (gosso22) [inline opensrp-chw/src/at/assets/json.form/pnc_hv_medication_in_use_sw.json]: Same comment here

**Review by gosso22 — COMMENTED** [review]
I left a few minor comments which we can address now or do a bit of research as far as data is concerned.

**Pull request merged**